pkg/controllers/deprovisioning/emptymachineconsolidation.go

```diff
@@ -80,9 +80,8 @@ func (c *EmptyMachineConsolidation) ComputeCommand(ctx context.Context, candidat
 	// the machine isn't a target of a recent scheduling simulation
 	for _, n := range candidatesToDelete {
 		if len(n.pods) != 0 && !c.cluster.IsNodeNominated(n.Name()) {
-			return Command{action: actionRetry}, nil
+			return Command{}, fmt.Errorf("command is no longer valid, %s", cmd)
```

Comment:
This will be bubbled up as "could not deprovision, command is no longer valid: command", which makes sense to us since we're deep in this code. I think we should include some mention of the Consolidation TTL. Can you reword it something like the following?
```suggestion
			return Command{}, fmt.Errorf("command is no longer valid after Consolidation TTL Validation, %s", cmd)
```

Reply:
```
    logger.go:130: 2023-04-10T14:30:33.217-0700	ERROR	deprovisioning/controller.go:121	Failed during "consolidation", computing deprovisioning decision, command is no longer valid, delete, terminating 1 machines diveviolet-21-2z1wh3y1dk/64-cpu-128-mem-arm64-windows-test-zone-3-on-demand/on-demand
```

Reply:
I don't think Consolidation TTL is part of this question.